main.go

```diff
@@ -27,20 +28,18 @@ func main() {
 
 	if len(os.Args) <= 1 {
 		plugin.FlagSet.Usage()
-		check.Exit(check.Unknown, "No arguments given")
+		check.ExitError(errors.New("No arguments given"))
```

Comment:
You can use an `check.ExitRaw(check.Unknown, "No arguments given")`, since the Plugin didn't do it's function properly at this point we tend to exit UNKNOWN since the status of whatever couln't be determined.

Reply:
According to the docs, `ExitError` exits with an `Unknown` state as well - do you want me to change it to `ExitRaw` anyways for semantics on the code level?

Reply:
I would vote for `check.ExitRaw(check.Unknown, "No arguments given")`

Reply:
I'll push the changes discussed in the conversations later today.